pages/linux/ip-route.md

```diff
@@ -0,0 +1,32 @@
+# ip route
+
+> IP Routing table management subcommand.
+> More information: <https://www.man7.org/linux/man-pages/man8/ip-route.8.html>.
+
+- Display the routing table:
+
+`ip route show/list`
+
+- Add a default route using gateway forwarding:
+
+`sudo ip route add default via {{gateway_ip}}`
+
+- Add a default route using eth0:
+
+`sudo ip route add default dev{{eth0}}`
+
+- Add a static route:
+
+`sudo ip route add {{destination_network_ip}} via {{gateway_ip}} dev {{eth0}}`
+
+- Delete a static route:
+
+`sudo ip route del {{destination_network_ip}} dev {{eth0}}`
+
+- Change/Replace a static route:
+
+`sudo ip route change/replace {{destination_network_ip}} via {{gateway_ip}} dev {{eth0}}`
+
+- Show which route will be used by the kernel to reach one ip address:
```

Comment:
```suggestion
- Show which route will be used by the kernel to reach a ip address:
```

Reply:
it's **an ip address**. 
ref: An Internet Protocol address (IP address)